ticdc/ticdc-changefeed-overview.md

```diff
@@ -35,6 +35,7 @@ The numbers in the preceding state transfer diagram are described as follows.
 - ⑥ The changefeed encounters an unrecoverable error and directly enters the failed state. At this time, the changefeed continues to block upstream GC for a duration specified by `gc-ttl`.
 - ⑦ The replication progress of the changefeed reaches the value set by `target-ts`, and the replication is completed.
 - ⑧ The changefeed has been suspended for a duration longer than the value specified by `gc-ttl`, thus encountering GC advancement errors, and cannot be resumed.
```

Comment:
Please update line 22 as well, which indicates the failed task cannot be resumed.

![image](https://github.com/pingcap/docs/assets/79440533/63bfc0db-071c-41cb-9c72-2216d01dbc03)

Reply:
Thank you for the comment @qiancai
The change in https://github.com/pingcap/docs/pull/15555/commits/66e7ea3cd45c0a377f40370fbf40b1e14847066a should address this feedback